packages/nyc-config-typescript/index.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extension": [
+    ".ts",
+    ".tsx"
+  ],
+  "exclude": [
```

Comment:
`exclude` does not get merged with the default config (unfortunately).  So I think you should add the default excludes from test-exclude:
```js
    'coverage/**',
    'packages/*/test/**',
    'test/**',
    'test{,-*}.js',
    '**/*{.,-}test.js',
    '**/__tests__/**',
    '**/node_modules/**',
    '**/babel.config.js'
```